### ISSUE TITLE
Too many open files (regression?)

### ISSUE THREAD

**Issue by WolfgangFahl** [ad-freiburg/qlever-control#4]
a few hours into the indexing after starting the  
```
INFO:  Merging partial vocabularies in byte order (internal only) ...
```
phase the indexing crashes with:
```
2022-05-23 00:10:53.144	- ERROR: ! ERROR opening file "wikidata.tmp.for-prefix-compression..tmp.partial-vocabulary.1018" with mode "r" (Too many open files)
```
see also https://github.com/ad-freiburg/qlever/issues/571

As far as i remember the script https://github.com/ad-freiburg/qlever/issues/562 sets the soft ulimit to 1048576 to avoid this but i am not sure whether this is a regression in the script or a bug (regression) in qlever itself.

**Comment by WolfgangFahl** [comment]
```bash
ulimit -Sn 1048576
```

**Comment by WolfgangFahl** [comment]
The error showed up again on the next attempt.
```bash
ulimit -n 1000000
```

**Comment by hannahbast** [comment]
Yes, very good point, this should definitely be part of the script.

**Comment by hannahbast** [comment]
Addressed by 3b5dbcb1fbe342b43ae460ddd04a4ea22be452b1
